homeassistant/components/plex/media_player.py

```diff
@@ -622,10 +623,24 @@ def device_info(self):
 
     async def async_browse_media(self, media_content_type=None, media_content_id=None):
         """Implement the websocket media browsing helper."""
+        is_internal = is_internal_request(self.hass)
         return await self.hass.async_add_executor_job(
             browse_media,
-            self.entity_id,
-            self.plex_server,
+            self,
+            is_internal,
             media_content_type,
             media_content_id,
         )
+
+    async def async_get_browse_image(
+        self, media_content_type, media_content_id, media_image_id=None
+    ):
+        """Get media image from Plex server."""
+        image_url = self.plex_server.thumbnail_cache.get(media_image_id)
```

Comment:
```suggestion
        image_url = self.plex_server.thumbnail_cache.get(media_content_id)
```

Reply:
So thumbnail_cache seems like an append only affair. Should that be changed?

Reply:
It was intentional as the item IDs are static and permanent. However, I suppose there's no downside to always updating the cache with the latest URL in case the user updates the media image in Plex.